SampleApp/build.gradle

```diff
@@ -17,13 +19,11 @@ repositories {
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 25
-    buildToolsVersion "25.0.2"
-
     defaultConfig {
         applicationId "io.card.development"
+        compileSdk 34
         minSdkVersion 18
-        targetSdkVersion 25
+        targetSdkVersion 34
```

Comment:
We can set the `targetSdkVersion` to 35 yeah; I think I didn't do it because this Android SDK has the edge-to-edge issue in Android 15 - the fix was done on the [cordova plugin](https://github.com/OutSystems/cardio-cordova-plugin/pull/8), not here.

As for setting `compileSdk` to 35, the project wasn't compiling and the build error didn't say much (just `NullPointerException`), and since these are the properties for the sample app, didn't bother much with it.

Reply:
Yeah we should be fine since these are properties of the Sample App.